alchemist/alchemist-incarnation-protelis/src/main/kotlin/it/unibo/alchemist/model/implementations/nodes/CustomNode.kt

```diff
@@ -0,0 +1,7 @@
+package it.unibo.alchemist.model.implementations.nodes
+
+import it.unibo.alchemist.model.interfaces.Environment
+
+class CustomNode<T>(env: Environment<T, *>) : AbstractNode<T>(env) {
+    override fun createT(): T = TODO()
```

Comment:
What is this class for?

Reply:
When i had doubts about how to load custom classes inside a simulation sice there wasn't written on the YAML doc how to do that, you asked me to insert a new paragraph and create a test inside protelis incarnation to do so.

Reply:
Then this one must get moved inside `test`, it's not part of the `main` sourceset.